### PR TITLE
LPS-179520

### PR DESCRIPTION
**Pull request by gabrielwas**
Related issue: https://issues.liferay.com/browse/LPS-179520

**Context**: We need to check the right permissions while performing aggregation facets. We've refactored the way we build the permissions predicate for listing entries, but we haven't changed it for the aggregation facet query.

**Fix**: Calling the right method to build the permissions predicate.

### CODE REVIEW

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java [hunk 1]
```diff
@@ -1256,6 +1258,69 @@ public void testGetObjectEntries() throws Exception {
...
+
+		Facet facet = facets.get(0);
+
+		List<Facet.FacetValue> facetValues = ListUtil.filter(
+			facet.getFacetValues(),
+			facetValue -> Objects.equals(facetValue.getTerm(), "Able"));
+
+		if (ListUtil.isEmpty(facetValues)) {
+			Assert.fail();
+		}
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java:1319]: @gabrielwas I've neve see this pattern for this case I believe assertFalse is more appropriate:

````java
Assert.assertFalse(ListUtil.isEmpty(facetValues));
````

what do you think?

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java [hunk 2]
```diff
@@ -1256,6 +1258,69 @@ public void testGetObjectEntries() throws Exception {
...
+			null,
+			_filterPredicateFactory.create(
+				null, _objectDefinition1.getObjectDefinitionId()),
+			null, null);
+
+		List<Facet> facets = page.getFacets();
+
+		if (ListUtil.isEmpty(facets)) {
+			Assert.fail();
+		}
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java:1309]: same here

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java (hunk 1 above)
Comment (gabrielwas) [reply modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java:1319]: Good catch! Thx!

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by MarinhoFeliphe** [comment]
LGTM!! :tada:

**Comment by gabrielwas** [comment]
ci:test:object

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:object

**Comment by gabrielwas** [comment]
ci:stop

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:object

**Comment by gabrielwas** [comment]
ci:forward

**Pull request closed without merging**